options.html

```diff
@@ -30,11 +30,25 @@
 							<br />
 							<input type="checkbox" id="minihelp" name="minihelp">
 							Show mini help
+							<br />
+							<br />
+							Show popup with delay: 
+							<input type="number" id="popupDelay" name="popupDelay" min="1"> milliseconds
 							</p>
 					</div>
 					<!-- <div id="keycon" class="tabcontent hide"> -->
 					<div id="keycon" class="tabContent">
 					<h1>Keyboard</h1>
+						<input type="checkbox" id="showOnKeyFlag" name="showOnKeyFlag" disabled>
+						Immediately show only when key <select id="showOnKey" name="showOnKey"> 

```

Comment:
I see.  Perhaps we should explicitly mention that it overrides the popup delay setting; at least for me I didn't make the connection.

Also, it seems like no delay is correct for the initial press of the modifier key, but if the user is holding down the modifier key while scanning a sentence, won't they want the same delay as a normal user?

I assume you played with this during implementation and it felt better?